zapier/introduction/zaps-ii/zapier-data.md

```diff
@@ -0,0 +1,37 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+type: normal
+
+---
+
+# Zapier Data
+
+---
+## Content
+
+Until now, we have only used simple actions in our zaps (monitoring some data or sending a message). Zapier is not limited to that, with another popular application being finding data and using it at a later time.
+
+We have somewhat done this, when we used the filter step to verify that someone has actually gained access to our base.
+
+Now, what if we want to know who has gained access to our base? This is entirely possible, and since we are using Google Sheets we don't require any extra steps.
+
+All that you would have to do is go to the Slack step, and in the `'Message Text'` box you can already access all the information seen in the linked spreadsheet.
```

Comment:
```suggestion
All that you would have to do is go to the Slack step. In the `'Message Text'` box you'll have access all the information related toany row change seen in the linked spreadsheet.
```

Reply:
I think it's worth sharing the syntax they'll use to access a variable (i.e. step1.field_name or how it looks exactly)

Reply:
from what I know the only way of accessing variables is by selecting them from the drop-down menu